sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/ApiClientConfigurationTest.java

```diff
@@ -15,18 +18,23 @@
 
 import com.codahale.metrics.MetricFilter;
 import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import java.util.List;
 import javax.ws.rs.core.Response;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junitpioneer.jupiter.SetSystemProperty;
 import org.sdase.commons.client.jersey.test.ClientTestApp;
 import org.sdase.commons.client.jersey.test.ClientTestConfig;
 import org.sdase.commons.client.jersey.test.MockApiClient;
 import org.sdase.commons.client.jersey.test.MockApiClient.Car;
 import org.sdase.commons.client.jersey.wiremock.testing.WireMockClassExtension;
+import org.sdase.commons.server.dropwizard.metadata.DetachedMetadataContext;
+import org.sdase.commons.server.dropwizard.metadata.MetadataContext;
 
 /** Test that http client configuration is correct. */
+@SetSystemProperty(key = "METADATA_FIELDS", value = "token-id")
```

Comment:
I hope that the name "token-id" is not confusing because it's close to the "Trace-Token"

Reply:
Damn … should be `tenant-id`